### PR TITLE
Slides for News-September-2023

### PR DESCRIPTION
**Pull request by pt247**
The only thing left is to update upcoming events.

### CODE REVIEW

File: 2023-09-06_PyData-News-September-2023/custom.css
Comment (kynan) [inline 2023-09-06_PyData-News-September-2023/custom.css:1]: Not referenced?

File: 2023-09-06_PyData-News-September-2023/styles/custom.css
Comment (kynan) [inline 2023-09-06_PyData-News-September-2023/styles/custom.css:1]: Not referenced?

**Review by kynan — CHANGES_REQUESTED** [review]
I think you can remove those CSS files?

**Comment by kynan** [comment]
@pt247 ?

File: 2023-09-06_PyData-News-September-2023/styles/custom.css
Comment (pt247) [reply 2023-09-06_PyData-News-September-2023/styles/custom.css:1]: Removed.

**Review by kynan — APPROVED** [review]

**Pull request merged**